common/locales/de.json

```diff
@@ -59,6 +59,14 @@
     "contentType": {
       "name": "Inhalttyp | Inhalttypen"
     },
+    "day": {
+      "fields": {
+        "dayResponsibles": "Tageschefs"
```

Comment:
Wie wärs mit "Tagesverantwortliche" um aktuellen Gendering-Diskussionen aus dem Weg zu gehen? Vielleicht hat eCamp v3 ja wie schon eCamp v2 genügend Einfluss auf den gebräuchlichen Slang um das von Vornherein etwas zu entschärfen ;)

Reply:
Yes, good point. Fully agree. Fixed in https://github.com/ecamp/ecamp3/pull/2404/commits/4657afd5ece3c5aa6a7f0dba2488e3cc50e9c71b